src/main/java/com/minecolonies/coremod/event/ColonyTagCapabilityProvider.java

```diff
@@ -0,0 +1,48 @@
+package com.minecolonies.coremod.event;
+
+import com.minecolonies.api.colony.IColonyTagCapability;
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.world.chunk.Chunk;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.world.chunk.Chunk'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)